google-cloud-core/src/main/java/com/google/cloud/Timestamp.java

```diff
@@ -121,13 +121,13 @@ public static Timestamp now() {
   public static Timestamp of(java.sql.Timestamp timestamp) {
     int nanos = timestamp.getNanos();
 
-    // A pre-epoch timestamp will be off by one second because of the way that integer division
+    // A pre-epoch timestamp can be off by one second because of the way that integer division
     // works. For example, -1001 / 1000 == -1. In this case of timestamps, we want this result to be
     // -2. This causes any pre-epoch timestamp to be off by 1 second - fix this by adjusting the
```

Comment:
"adjusting": is this forward or backward? Do we add or subtract one second?

Reply:
Changed to 'subtract'.